packages/web-react/src/components/Dropdown/stories/Dropdown.tsx

```diff
@@ -0,0 +1,50 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context/IconsContext';
+import { Icon } from '../../Icon';
+import Dropdown from '../Dropdown';
+import { SpiritDropdownProps, DropdownRenderProps } from '../../../types';
+// import { Button } from '../../Button';
+
+const Story: ComponentStory<typeof Dropdown> = (args: SpiritDropdownProps) => (
+  <IconsProvider value={icons}>
+    <Dropdown {...args} />
+  </IconsProvider>
+);
+
+Story.args = {
+  id: 'DemoDropdown',
+  renderTrigger: ({ isOpen, trigger: { ...restOf } }: DropdownRenderProps) => (
+    // Missing 'ref' or not working 'innerRef' in Button component !!!
```

Comment:
Is there any issue to make this actionable?

Reply:
There is no issue no, but we must allow our components to use refs soon. I will create one.

Reply:
Cool, thank you!

Reply:
Nevertheless this is something like "TODO" isn't it? I would expect the line as follows (or am I wrong?)

`// TODO: Missing 'ref' or not working 'innerRef' in Button component !!!`